Huffman_Encoding/Huffman_Encoding using STL.cpp

```diff
@@ -0,0 +1,94 @@
+#include<bits/stdc++.h>
```

Comment:
Fix file name.

Reply:
What should be the name?

Reply:
Ohh sorry, I missed this.

Reply:
Huffman_Encoding_STL.cpp

Will be better.